packages/newsroom-signup/src/ApplyToTCR/index.tsx

```diff
@@ -12,7 +12,7 @@ export interface ApplyToTCRStepOwnProps {
   address?: EthAddress;
   newsroom: any;
   civil?: Civil;
-  navigate(go: 1 | -1): void;
+  navigate(go: 1 | -1, allDone?: boolean): void;
```

Comment:
can't devine this with like `step === numsteps`?

Reply:
or `step > numsteps`

Reply:
Yes, you are absolutely right (can just check `step = STEP.APPLIED`, simplifies the hack a bit.